sensor_control/navigator_battery_monitor/package.xml

```diff
@@ -14,8 +14,7 @@
   <run_depend>rospy</run_depend>
   <run_depend>std_msgs</run_depend>
   <run_depend>roboteq_msgs</run_depend>
-  <run_depend>sub8_alarm</run_depend>
+  <run_depend>navigator_alarm</run_depend>
```

Comment:
I was not sure about this. The battery monitor does import it, so it is technically a runtime dependency.